### PR TITLE
Layout fix

### PR DESCRIPTION
**Pull request by guergana**
Fixes to layout commit: 
- h1 font weight set to 400
- padding applied to anchor tabs instead of li element
- svg served as asset

### CODE REVIEW

File: resources/sass/app.scss [hunk 1]
```diff
@@ -93,7 +93,11 @@ nav.tabs {
             }
         
             a {
+                display: block;
                 color: #54595D;
+                padding: 4px 10px;
+                line-height: 24px;
+
```
Comment (Silvan-WMDE) [inline resources/sass/app.scss:100]: works great now!

File: resources/sass/app.scss [hunk 2]
```diff
@@ -79,7 +80,6 @@ nav.tabs {
         border-bottom: 1px solid #A2A9B1;
 
         li {
-            padding: 4px 10px;
             border: 2px solid white;
             transition-property: border;
```
Comment (Silvan-WMDE) [inline resources/sass/app.scss:84]:
```suggestion
            transition-property: border-bottom;
```
Without this change, my Chrome browser flashes the borders around the nav tabs upon reload. You can see it more clearly when you set the transition-duration to 1000ms (for testing).

**Review by Silvan-WMDE — COMMENTED** [review]
one last change - then it should be good. :-)

File: resources/sass/app.scss (hunk 2 above)
Comment (guergana) [reply resources/sass/app.scss:84]: Thanks for the suggestion. I will commit it as part of other changes I am adding.

Comment (guergana) [reply resources/sass/app.scss:84]: Seems the transitions firing onload is a Chrome bug. Applying [this solution](https://stackoverflow.com/questions/14389566/stop-css-transition-from-firing-on-page-load/42969608#42969608) works.

**Review by Silvan-WMDE — APPROVED** [review]
brilliant - works like a charm!

**Pull request merged**